homepage.css

```diff
@@ -144,54 +155,212 @@ ul li:hover ul{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images1 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images2{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images2 img{
```

Comment:
31/08/2021: Setting the height for the the img in the left side of the time that display the total duration.